srcpkgs/foot/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'foot'
 pkgname=foot
-version=1.14.0
+version=1.15.0
 revision=1
 build_style=meson
-configure_args="-Dwerror=false -Ddefault-utempter-path=/usr/lib/utempter/utempter"
+configure_args="-Dwerror=false -Dutmp-default-helper-path=/usr/lib/utempter/utempter"
```

Comment:
we might be able to drop this altogether:

- https://codeberg.org/dnkl/foot/src/branch/master/meson.build#L23
- https://codeberg.org/dnkl/foot/src/branch/master/meson.build#L43

Reply:
utmp logging still works without it